src/api/tests/service/profile/test_profile_routes.py

```
@@ -0,0 +1,79 @@
+
+
+@pytest.mark.usefixtures("app", "client")
+class TestProfileRoutes:
+    def test_hide_unused_profile_items_requires_parent(self, client):
+        resp = client.post("/api/profiles/hide-unused-profile-items", json={})
+        assert resp.status_code == 400
+
+    def test_hide_unused_profile_items_ok(self, monkeypatch, client, app):
+        called = {}
+
+        def fake_hide(app_ctx, parent_id, user_id):
```

Ruff (0.14.13)

Unused function argument: `app_ctx`

(ARG001)